### PR TITLE
Update broken link

### PR DESCRIPTION
**Pull request by alanyee**
-Updated 404 link to an updated link
-Made a few implicit HTTP calls explicit

### CODE REVIEW

File: pyguide.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<HTML xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcq="http://purl.org/dc/qualifiers/1.0/" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:fn="http://www.w3.org/2005/xpath-functions">
+<HTML xmlns:rdf="https://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcq="http://purl.org/dc/qualifiers/1.0/" xmlns:fo="https://www.w3.org/1999/XSL/Format" xmlns:fn="https://www.w3.org/2005/xpath-functions">
```
Comment (ejona86) [inline pyguide.html]: Namespaces don't work that way. It should be exactly what it was, with http.

Comment (alanyee) [inline pyguide.html]: Reverted those changes

**Comment by alanyee** [comment]
@eglaysher I think this PR has been left open for long enough.

**Comment by alanyee** [comment]
@ejona86 can you merge this PR?

**Review by ejona86 — APPROVED** [review]

**Pull request merged**